### ISSUE TITLE
Add external.input.video

### ISSUE THREAD

**Issue by smimram** [savonet/liquidsoap#284]
Now that there is native AVI support, see #256, we should add `input.external.video` (and the same for output), so that the following would work for instance:
```
s = input.external.video("ffmpeg -f lavfi -i testsrc -t 5 -f avi pipe:1")
output.sdl(faillible=true,s)
```
Right now `input.external` only supports audio.

**Comment by toots** [comment]
Whoa. I totally missed the avi encoder PR, nice!

**Comment by smimram** [comment]
Thanks! It won't be for next release, though.

**Comment by smimram** [comment]
This should be fixed with the merge 68ddd47ac17bbda082532447bbf070aba01403ea.
